techAssignment/src/components/InputCommand.vue

```diff
@@ -0,0 +1,70 @@
+<script>
+export default {
+  name: "InputCommand",
+  props: {
+    propCommandInput: {
+      type: String,
+      default: "",
+    },
+    propSetCommandInput: {
+      type: Function,
+      default(saveInput) {
+        return saveInput;
+      },
+    },
+    renderCommand: {
+      type: Function,
+      default() {
+        return {};
+      },
+    },
```

Comment:
I think we should use [Component events (emits) ](https://vuejs.org/guide/components/events.html#emitting-and-listening-to-events ) em:
- `propCommandInput`, `propSetCommandInput`: use [v-model](https://vuejs.org/guide/components/events.html#usage-with-v-model)
- `renderCommand` should be an event (could be called: `submit`). So instead of `this.renderCommand()` we could use: `this.$emit('submit')`